### DISCUSSION TITLE
_id for database

### DISCUSSION THREAD

**Discussion by DesignrKnight** [dscnitrourkela/project-avocado-web#38] [Q&A] [answered]
@riteshsp2000 What say we use the institute roll number as _id. That way linking between collections will be easier with the present data in Excel

**Comment by riteshsp2000** [comment, marked as answer]
This could help actually. I guess for graphql to get any mentor/mentee we need their id, maintaining _id and rollnumber might be a hassle

**Reply by DesignrKnight** [reply]
Let us maintain both for scalability later. But for now, we will have value same in both field
